sdcm/utils/adaptive_timeouts/__init__.py

```diff
@@ -73,6 +73,14 @@ class Operations(Enum):
                                  ("timeout", "service_level_for_test_step"))
 
 
+class TestInfoServices:  # pylint: disable=too-few-public-methods
+    @staticmethod
+    def get(node: "BaseNode") -> dict:
+        return dict(
+            n_db_nodes=len(node.parent_cluster.nodes)
```

Comment:
Why not taking it from the params?

Reply:
I thought sending out the current situation, would be better here.
but it's not critical, can get it from the sct config.